crypto/x509/x509_lu.c

```diff
@@ -886,18 +890,19 @@ void X509_STORE_set_check_issued(X509_STORE *ctx,
     ctx->check_issued = check_issued;
 }
 
-X509_STORE_CTX_check_issued_fn X509_STORE_get_check_issued(const X509_STORE *ctx)
+X509_STORE_CTX_check_issued_fn X509_STORE_get_check_issued(const X509_STORE *cx)
 {
-    return ctx->check_issued;
+    return cx->check_issued;
 }
 
 void X509_STORE_set_check_revocation(X509_STORE *ctx,
-                                     X509_STORE_CTX_check_revocation_fn check_revocation)
+                                     X509_STORE_CTX_check_revocation_fn cb)
 {
-    ctx->check_revocation = check_revocation;
+    ctx->check_revocation = cb;
 }
 
-X509_STORE_CTX_check_revocation_fn X509_STORE_get_check_revocation(const X509_STORE *ctx)
+X509_STORE_CTX_check_revocation_fn
+X509_STORE_get_check_revocation(const X509_STORE *ctx)
```

Comment:
Why changing ctx above and not here?

Reply:
> Why changing ctx above and not here?

Because here the line anyway had to be split, and after that there is enough space for the usual name `ctx`.

Reply:
I would not split the line. The 80 characters limit is not a hard one.

Reply:
Nice, but then the nuisance is that `check-format.pl` will complain.

Reply:
Anyway, reverted splitting the line here.